src/Sarif/Core/Invocation.cs

```diff
@@ -52,6 +52,8 @@ public partial class Invocation
                 invocation.ExecutableLocation.Uri = new Uri(assembly.Location);
             }
 
+            invocation.ExecutionSuccessful = true;
```

Comment:
>invocation.ExecutionSuccessful = true; [](start = 12, length = 38)

i am unsure if this is valid assignment! i couldnt figure out how to set this right, please cross check

Reply:
this is fine for now. the thing actually driving tool execution needs to reset this to false if required.

---
In reply to: [279147950](https://github.com/Microsoft/sarif-sdk/pull/1465#discussion_r279147950) [](ancestors = 279147950)